README.md

```diff
@@ -21,7 +21,7 @@ for more information.
 
 Notary aims to make the internet more secure by making it easy for people to
 publish and verify content. We often rely on TLS to secure our communications
-with a web server which is inherently flawed, as any compromise of the server
+with a web server, which is inherently flawed, as any compromise of the server
```

Comment:
I think in this case `which is inherently flawed` is a essential clause and doesn't need the comma.

Reply:
Hi there, thanks for contributing to the docs!

I'm not a native speaker and I can hardly tell which is the right one.

What we want to tell people is, the old-ways(only rely on TLS) to secure our communication is inherently flawed.

Reply:
Indeed. To communicate that clearly and correctly, the comma would be necessary.

Reply:
LGTM once Gordon stop shouting